### PR TITLE
Added mailsmtp_data_message_quit_no_disconnect interface

### PR DESCRIPTION
**Pull request by AlexKar**

### CODE REVIEW

File: src/low-level/smtp/mailsmtp.c [hunk 1]
```diff
@@ -489,6 +489,43 @@ int mailsmtp_data_message_quit(mailsmtp * session,
     }
 }
 
+int mailsmtp_data_message_quit_no_disconnect(mailsmtp * session,
```
Comment (dinhviethoa) [inline src/low-level/smtp/mailsmtp.c]: Could you reuse that function in mailsmtp_data_message_quit()?

File: src/low-level/smtp/mailsmtp_helper.c [hunk 2]
```diff
@@ -160,6 +160,53 @@ int mailesmtp_send_quit(mailsmtp * session,
 }
 
 LIBETPAN_EXPORT
+int mailesmtp_send_quit_no_disconnect(mailsmtp * session,
```
Comment (dinhviethoa) [inline src/low-level/smtp/mailsmtp_helper.c]: Could you use that function in mailesmtp_send_quit()?

File: src/low-level/smtp/mailsmtp.c (hunk 1 above)
Comment (AlexKar) [inline src/low-level/smtp/mailsmtp.c]: done

File: src/low-level/smtp/mailsmtp_helper.c (hunk 2 above)
Comment (AlexKar) [inline src/low-level/smtp/mailsmtp_helper.c]: done

**Pull request merged**